gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/CredentialsController.groovy

```diff
@@ -120,7 +120,7 @@ class CredentialsController {
 
   @PutMapping
   @ApiOperation('Updates an existing account definition.')
-  @PreAuthorize("hasPermission(#definition.name, 'ACCOUNT', 'WRITE')")
+  @PreAuthorize("hasPermission(#accountDefinition.name, 'ACCOUNT', 'WRITE')")
```

Comment:
I gotta ask if it's possible/easy to write a test that exposes this.

Reply:
In Java files, IntelliJ throws warnings at you when you try to reference things in SpEL annotations that don't exist. The Groovy version doesn't do that, however. A test could be possible, though it's mostly setting up test fixtures and verifying said fixture works as expected.

Reply:
Maybe one day this'll become java...